packages/suite/src/hooks/suite/useLoadingSkeleton.ts

```diff
@@ -0,0 +1,13 @@
+import { useSelector } from '@suite-hooks';
+
+export const useLoadingSkeleton = () => {
+    const waitingForDevice = !useSelector(state => state.suite.device)?.state;
```

Comment:
It is possible to have device without state in routes that do not trigger discovery, for example settings. Therefore I am not sure whether this is 100% bulletproof

Reply:
ohh, I didn't realized that. Right now we are good since there are no skeletons in settings 😁 . Another thing is, the usage of this hook is not mandatory, there is still an option to provide your custom `shouldAnimate` prop (this is actually used on dashboard where we don't care we already have device's state because we want to show loadings untill all accounts are loaded). This is just a shortcut for most obvious usage which could be expanded later.

But good thing you pointed it out we can add some more logic to it when it'll be necessary.

Reply:
Oook, I couldn't come up with a better idea myself..